src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -851,7 +851,7 @@ private void LoadSamplesMenu()
                 {
                     var showInFolder = new MenuItem
                     {
-                        Header = "Show In Folder",
+                        Header = Wpf.Properties.Resources.DynamoViewHelpMenuShowInFolder,
```

Comment:
This one is [here](https://github.com/DynamoDS/Dynamo/blob/localization/src/DynamoCoreWpf/Properties/Resources.resx#L799) .

For the other two: [1](https://github.com/nguyen-binh-minh/Dynamo/blob/tagging/src/DynamoCoreWpf/Properties/Resources.resx#L1477), [2](https://github.com/nguyen-binh-minh/Dynamo/blob/tagging/src/DynamoCoreWpf/Properties/Resources.resx#L1480)

I don't know why they don't show up in the diff.
@Benglin PTAL